terraform/aws/iglu_server/default/README.md

```diff
@@ -10,7 +10,7 @@
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | ~> 3.25.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 3.25.0 |
```

Comment:
This is because you have a terraform lock file in here when you generated the read me. You need to remove the lockfile before generating the read me otherwise if this is used directly as a module you have to use exactly this version of AWS.